catalog/async/catalog-async-data-api/src/main/java/org/codice/ddf/catalog/async/data/api/internal/ProcessResource.java

```diff
@@ -87,4 +87,11 @@
      * @return {@code true} if modified, {@code false} otherwise
      */
     boolean isModified();
+
+    /**
+     * This is used to close the input stream.
+     * <p>
+     * Look into having the ProcessResource implement InputStream
```

Comment:
Could also consider this to support try with resources: https://docs.oracle.com/javase/7/docs/api/java/io/Closeable.html